state/secrets/get.go

```diff
@@ -53,11 +54,11 @@ func (cmd *Command) ExecuteGet(_ *cobra.Command, args []string) {
 			if secret.IsUser() {
 				err = "secrets_err_user_not_defined"
 			}
-			print.Error(locale.Tr(err, cmd.Args.Name))
+			fmt.Fprint(os.Stderr, locale.Tr(err, cmd.Args.Name))
```

Comment:
Can't we use the `outputer` library in this file?

Reply:
Secrets is entirely legacy, using outputer would require serious refactoring. I'd rather do this in a separate story in one go.